src/modules/statusTask.js

```diff
@@ -0,0 +1,22 @@
+import Tasks from './classTasks.js';
+
+const statusTask = () => {
+  const checkboxes = document.querySelectorAll('.checkbox');
+  checkboxes.forEach((checkbox) => {
+    checkbox.addEventListener('change', (e) => {
+      const { target } = e;
+      const targetId = +target.parentNode.parentNode.className - 1;
+      if (target.checked) {
+        target.nextSibling.classList.add('finished');
+        Tasks.tasks[targetId].completed = true;
+        Tasks.storage(Tasks.tasks);
+      } else {
+        target.nextSibling.classList.remove('finished');
+        Tasks.tasks[targetId].completed = false;
+        Tasks.storage(Tasks.tasks);
+      }
+    });
+  });
+};
```

Comment:
I recommended you to use toggle instead of add or remove so as not to repeat the same lines of code and you would save more lines of code, do you agree?

`const statusTask = () => {
  const checkboxes = document.querySelectorAll('.checkbox');
  checkboxes.forEach((checkbox) => {

    checkbox.addEventListener('change', (e) => {

      const { target } = e;
      const targetId = +target.parentNode.parentNode.className - 1;

      target.nextSibling.classList.toggle('finished');
      Tasks.tasks[targetId].completed = !Tasks.tasks[targetId].completed;
      Tasks.storage(Tasks.tasks);

      });
  });
}; 
`

Reply:
Wow! Great, thanks for this suggestion, I will implement it.